server/ladder_service.py

```diff
@@ -259,10 +257,10 @@ def _cancel_search(self, initiator: Player, queue_name: str) -> None:
             "%s stopped searching for %s", cancelled_search, queue_name
         )
 
-    def inform_player(self, player: Player):
+    def inform_player(self, player: Player, rating_type: str) -> None:
```

Comment:
TBH I think this could have a more descriptive name. Maybe `notify_about_initial_learning_phase` (which is the shortest I could think of in one minute).

Reply:
`send_rating_progress_msg`?

Reply:
IMO this function shouldn't exist at all :D, it should be done in the client, but yea that name sounds better